src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -234,4 +237,32 @@ void testIsPaperCaseAttending() {
 
         assertThat(result).isFalse();
     }
+
+    @DisplayName("Return Adjourn case type of next hearing when value present")
+    @Test
```

Comment:
Expand this to parameterized to take all three possible values?

Reply:
done